server/controllers/index.js

```diff
@@ -4,9 +4,11 @@ const { getPlaces } = require('./places')
 const { login } = require('./login')
 const { getGuides } = require('./guides')
 const { isAuthenticated } = require('../middleware/auth')
+const { getArabicWords } = require('./arabicList')
 
 router.get('/places', isAuthenticated, getPlaces)
 router.post('/login', login)
 router.get('/guides', getGuides)
+router.get('/arabic-words', getArabicWords)
```

Comment:
Does someone need to be authenticated to see this?

Reply:
No, just we need an authentication when the tourist want to add a review for a guides or places.